mosaic_creator.py

```diff
@@ -53,7 +69,7 @@ def getBestMatchIndex(input_avg, avgs):
             min_index = index
         index += 1
     return (min_index)
-# creating the image grid for tiles to fit in
+# creating the image grid for tilehttps://twitter.com/notificationss to fit in
```

Comment:
What's this about😅

Reply:
OMG! I'm sorry for this mistake, I didn't realize it. Thanks for having patience and for helping me to improve my skills.